transmart-api-server/build.gradle

```diff
@@ -35,3 +35,11 @@ dependencies {
 bootRun {
     jvmArgs = ['-Dspring.output.ansi.enabled=always']
 }
+
+publishing {
+    publications {
+        mavenJava(MavenPublication) {
```

Comment:
The result of this is not executable with `java -jar`.
We need something like this (see [transmart-server/build.gradle](https://github.com/thehyve/transmart-core/blob/dev/transmart-server/build.gradle#L86)):
```gradle
task executableWar(type: Copy) {
    from "${libsDir}/${project.name}-${project.version}.war"
    into "${distsDir}"
}
executableWar.dependsOn 'bootRepackage'

publishing {
    publications {
        mavenJava(MavenPublication) {
            artifact file("${distsDir}/${project.name}-${project.version}.war") // Publish the output of the executableWar task
        }
    }
}
publishToMavenLocal.dependsOn 'executableWar'
publish.dependsOn 'executableWar'
```